test/framework/integration_framework/iroha_instance.cpp

```diff
@@ -52,17 +56,30 @@ namespace integration_framework {
     instance_->storage->insertBlock(block);
   }
 
+  void IrohaInstance::setMstGossipParams(
+      std::chrono::milliseconds mst_gossip_emitting_period,
+      uint32_t mst_gossip_amount_per_once) {
+    BOOST_ASSERT_MSG(
+        !instance_,
```

Comment:
```suggestion
        not instance_,
```
since this notation is mostly used instead in the project.